gooey/cli.py

```diff
@@ -0,0 +1,41 @@
+'''The command line interface.'''
+
+import sys
+from importlib import import_module
+try:
+    from importlib.metadata import entry_points
```

Comment:
Explain a bit about these different versions?

Reply:
`importlib.metadata` is included in the standard library since Python 3.8 only.
Since you support lower versions, the `importlib-metadata` package is required (it's a simple backport).
Therefore, we must handle both cases in the code since the import path is not the same (`importlib.metadata` vs. `importlib_metadata`).